pyomo/contrib/pynumero/algorithms/solvers/tests/test_pyomo_ext_cyipopt.py

```diff
@@ -179,6 +179,7 @@ def test_pyomo_external_model_scaling(self):
 
         with open('_cyipopt-pyomo-ext-scaling.log', 'r') as fd:
             solver_trace = fd.read()
+        cyipopt_problem.close()
         os.remove('_cyipopt-pyomo-ext-scaling.log')
```

Comment:
This call to `cyipopt.Problem.close` is now necessary on windows if we try to remove a file too early (I think before the garbage collector has deallocated the `cyipopt.Problem`, but my testing was inconclusive).

This is another way this PR could break a user's code.